packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -245,7 +245,20 @@ const makeRenderer = (
 					initialNow: new Date().getTime(),
 					isProdApi: server.settings.app.api.isProd,
 					isQL: parseMemberCookie(state).ql === 'true',
-					memberId: parseMemberCookie(state).id,
+					/**
+					 * Original tickets that caused need:
+					 * 
+					 * @see https://meetup.atlassian.net/browse/MG-2473
+					 * @see https://meetup.atlassian.net/browse/MG-2441
+					 * 
+					 * For segmentation and rollout purposes, it is extremely advantageous
+					 * to have the member object available pre query to do split tests
+					 * based off of things like the member's id or location.
+					 * 
+					 * If the member is not logged in, this will return an empty object
+					 * with a member id of 0.
+					 */
+					member: parseMemberCookie(state),
```

Comment:
if you're removing `memberId`, then this PR is introducing a breaking change - I recommend leaving `memberId` as a standalone param for now, and potentially adding a `// deprecated - use member.id` comment to indicate that it should not be referenced. Since you're defining a new 'correct' place to look for `member.id`, make sure you look for and update places where `config.memberId` is currently being used elsewhere in the platform (e.g. https://github.com/meetup/meetup-web-platform/blob/master/packages/mwp-api-state/src/cache/index.js#L33)

You'll also want to update [this line](https://github.com/meetup/mup-web/blob/master/src/app/root/appSelectors.js#L103) in mup-web

Finally, you'll need to add a new type for `config.member` to the `MWPState` type: https://github.com/meetup/meetup-web-platform/blob/master/flow-typed/platform.js#L10